site/gatsby-site/blog/incident-report-2022-october/index.fr.mdx

```diff
@@ -1,143 +1,142 @@
 
-**Incident #383:** [Google Home Mini Speaker Reportedly Read N-Word in Song Title Aloud](/cite/383)
+**Incident #383 :** [Le mini haut-parleur Google Home aurait lu à voix haute N-Word dans le titre de la chanson](/cite/383)
 
-* **What happened?** Google Home Mini speaker was reported by users for announcing aloud the previously-censored n-word in a song title. It is unclear when or how Google's speakers stopped censoring the n-word.
-* **Who was involved?** [Google Home](/entities/google-home) developed and deployed an AI system, which harmed [Black Google Home Mini users](/entities/black-google-home-mini-users) and [Google Home Mini users](/entities/google-home-mini-users).
+* **Que s'est-il passé ?** Le haut-parleur Google Home Mini a été signalé par des utilisateurs pour avoir annoncé à haute voix le mot n précédemment censuré dans un titre de chanson. On ne sait pas quand ni comment les haut-parleurs de Google ont cessé de censurer le n-word.
+* **Qui était impliqué ?** [Google Home](/entities/google-home) a développé et déployé un système d'IA, qui a nui aux [utilisateurs noirs de Google Home Mini](/entities/black-google-home-mini-users ) et [utilisateurs de Google Home Mini](/entities/google-home-mini-users).
 
-**Incident #384:** [Glovo Driver in Italy Fired via Automated Email after Being Killed in Accident](/cite/384)
+**Incident #384 :** [Un chauffeur Glovo en Italie a été licencié par e-mail automatisé après avoir été tué dans un accident](/cite/384)
 
-* **What happened?** Delivery company Glovo's automated system sent an email terminating an employee for "non-compliance terms and conditions" after the employee was killed in a car accident while making a delivery on Glovo's behalf.
-* **Who was involved?** [Glovo](/entities/glovo) developed and deployed an AI system, which harmed [Sebastian Galassi](/entities/sebastian-galassi) and [Sebastian Galassi's family](/entities/sebastian-galassi's-family).
+* **Que s'est-il passé ?** Le système automatisé de la société de livraison Glovo a envoyé un e-mail licenciant un employé pour "non-respect des conditions générales" après que l'employé a été tué dans un accident de voiture alors qu'il effectuait une livraison au nom de Glovo.
+* **Qui était impliqué ?** [Glovo](/entities/glovo) a développé et déployé un système d'IA, qui a nui à [Sebastian Galassi](/entities/sebastian-galassi) et [à la famille de Sebastian Galassi](/entities/sebastian -famille de galassi).
```

Comment:
The family link is broken and should not be translated. i.e. `[à la famille de Sebastian Galassi](/entities/sebastian-galassi's-family)`